src/main/java/io/kestra/plugin/typesense/BulkIndex.java

```diff
@@ -51,33 +56,76 @@
         )
     }
 )
-public class BulkIndex extends AbstractTypesenseTask implements RunnableTask<VoidOutput> {
+public class BulkIndex extends AbstractTypesenseTask implements RunnableTask<BulkIndex.Output> {
 
     @Schema(
         title = "The file URI containing the documents to index"
     )
+    @NotNull
     private Property<String> from;
 
+    @Schema(
+        title = "The chunk size for every bulk request",
+        description = "Default value is 1000"
+    )
+    private Property<Integer> chunk;
```

Comment:
```suggestion
    private Property<Integer> chunk = Property.of(10);
```

This would correctly document the default value.